### PR TITLE
WIP: adding campaign support

### PR DESCRIPTION
**Pull request by mtam2**
TODO:
- update documentation
- add tests

### CODE REVIEW

File: lib/active_campaign/api/campaign_messages.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module ActiveCampaign
+  module API
+    #
+    # Interface to campaign endpoints
+    #
+    # @author Mikael Henriksson <mikael@mhenrixon.com>
```
Comment (rylwin) [inline lib/active_campaign/api/campaign_messages.rb]: @mtam2 I think you get to be the author of these changes if you want

File: lib/active_campaign/client.rb [hunk 2]
```diff
@@ -13,6 +13,8 @@ class Client
     endpoint :account_contacts
     endpoint :accounts
     endpoint :addresses
+    endpoint :campaigns
+    endpoint :campaign_messages
```
Comment (rylwin) [inline lib/active_campaign/client.rb:16]: swap the order of the new lines to maintain alpha order?

**Review by rylwin — APPROVED** [review]
Looks great! 2 minor comments for your consideration. :rocket: when ready!

**Comment by mtam2** [comment]
Awesome, I've reordered the endpoints, changed the author to me, and rebased this into 1 commit. Only you have permission to merge so I'll update the Gemfile after the merge.

**Pull request merged**